DigitalOcean.API/Models/Responses/CdnEndpoint.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace DigitalOcean.API.Models.Responses {
+    public class CdnEndpoint {
+        /// <summary>
+        /// A unique ID that can be used to identify and reference a CDN endpoint.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The fully qualified domain name (FQDN) for the origin server which the provides the content for the CDN.
+        /// This is currently restricted to a Space.
+        /// </summary>
+        public string Origin { get; set; }
+
+        /// <summary>
+        /// The fully qualified domain name (FQDN) from which the CDN-backed content is served.
+        /// </summary>
+        public string Endpoint { get; set; }
+
+        /// <summary>
+        /// A time value given in ISO8601 combined date and time format that represents when the CDN endpoint was created.
+        /// </summary>
+        public DateTime CreatedAt { get; set; }
+
+        /// <summary>
+        /// The amount of time the content is cached by the CDN's edge servers in seconds.
+        /// </summary>
+        public int TTL { get; set; }
```

Comment:
Can you confirm that this gets filled in from a response?

Off the top of my head, this may cause an issue due to the capitilization. In other response models it is listed as `Ttl`.

Reply:
I had considered that myself, since most of the time the json deserializers are only so smart. It definitely gets correctly filled for me. I wonder however if this might be from the version of Newtonsoft? If this was a particular problem in the past, that is.

The only reason I went with all caps is because the DomainRecord model also used all capital `TTL`. I hadn't noticed the Domain model used `Ttl`. I'm fine with switching to camel cased, it is a good consistency to keep in my opinion.

Reply:
I think swapping to `Ttl` is the way to go. I hadn't noticed the inconsistency with DomainRecord :(

Reply:
Easy fix.